include/openmc/mgxs_interface.h

```diff
@@ -12,36 +12,71 @@
 namespace openmc {
 
 //==============================================================================
-// Global variables
+// Global MGXS data container structure
 //==============================================================================
 
-namespace data {
+struct MgxsInterface
+{
```

Comment:
Style guide says brace should go on line with struct

Reply:
ok, fixed